web/vite.config.js

```diff
@@ -31,4 +31,7 @@ export default defineConfig({
       },
     },
   },
+  build: {
+    sourcemap: "inline",
```

Comment:
can we build these not inline? it balloons the size of index.js from 500kb to 5mb :sweat_smile:

Reply:
I thought they didn't show up if they weren't inline but I was wrong, seems to work and it's down to 555 kb on my end again